src/Microsoft.Azure.WebJobs.Host/Config/ExtensionConfigContextExtensions.cs

```diff
@@ -69,5 +71,77 @@ public static class ExtensionConfigContextExtensions
             IExtensionRegistry extensions = context.Config.GetService<IExtensionRegistry>();
             extensions.RegisterBindingRules<TAttribute>(bindingProviders);
         }
+
+        /// <summary>
```

Comment:
These are helpers for plumbing host.json through to the extension. 
Some extensions need just a specific section, some need the entire host.json (since they handle multiple sections). 
If the host.json is consistent with the config (same property names, etc), then the extension can just call the Apply() helper.